vividus-bdd-engine/src/main/java/org/vividus/bdd/expression/RoundExpressionProcessor.java

```diff
@@ -38,8 +38,8 @@
     {
         String roundingModes = Stream.of(RoundingMode.values()).map(e -> e.name().toLowerCase())
                 .collect(Collectors.joining("|"));
-        String pattern = String.format("^round(?:\\((-?\\d+(?:\\.\\d*)?)(?:,\\s*(\\d+))?(?:,\\s*(%s))?\\))$",
-            roundingModes);
+        String pattern = String.format("^round(?:\\((-?\\d+(?:\\.\\d*)?(?:[Ee]+-?\\d+)?)(?:,\\s*(\\d+))?"
```

Comment:
the pattern won't match `0.15237E+2` which is a valid number

Reply:
fixed